jekyll/_cci2/configuration-reference.md

```diff
@@ -1533,7 +1533,14 @@ workflows:
 
 ##### **Using `when` in Workflows**
 
-With CircleCI configuration v2.1, you may use a `when` clause (the inverse clause `unless` is also supported) under a workflow declaration with a truthy or falsy value to determine whether or not to run that workflow. The most common use of `when` is with CircleCI API v2 pipeline triggering with parameters.
+With CircleCI v2.1 configuration, you may use a `when` clause (the inverse clause `unless` is also supported) under a workflow declaration with a truthy or falsy value to determine whether or not to run that workflow. The most common use of `when` is with [CircleCI API v2 pipeline triggering](https://circleci.com/docs/api/v2/#trigger-a-new-pipeline) with parameters. Truthy/falsy calues can be booleans, numbers, and strings. Falsy would be any of: false, 0, empty string, nil, and NaN. Everything else would be truthy.
+
+Key | Required | Type | Description
```

Comment:
I need some help filling in this table 💥 😬

Reply:
I'm not quite sure where these rows come from?